### DISCUSSION TITLE
post/2024-google-cloud-next-1/

### DISCUSSION THREAD

**Discussion by giscus** [kakaopay/techlog-comments#82] [Announcements]
# post/2024-google-cloud-next-1/

Google Cloud Next 2024 주요 내용과 참관 후 우리가 준비해야 할 건 무엇일지 고민한 내용을 공유합니다.

https://tech.kakaopay.com/post/2024-google-cloud-next-1/

**Comment by trendnote** [comment]
단순히 구글 넥스트 소개인줄 알았는데, 좋은 인사리트를 주셔서 감사합니다. 
좋은 컨텐츠 잘 읽고 갑니다.

**Reply by pos-man** [reply]
좋게 봐주셔서 고맙습니다!
